packages/components/time_region_value/src/routes/[id]/index.svelte

```diff
@@ -463,216 +343,122 @@
-								>
-									<text dy={-tooltipShift} class='bkg'>{$tooltip.value}</text>
-									<text dy={-tooltipShift}>{$tooltip.value}</text>
-									<text dy={tooltipShift} class='bkg'>{$tooltip.nuts_label}</text>
-									<text dy={tooltipShift}>{$tooltip.nuts_label}</text>
-								</g>
-							</g>
-						{/if}
-
-					{:else}
-						<text
-							class='message'
-							x={width / 2}
-							y={height / 2}
-						>No data</text>
-					{/if}
-				</svg>
-			{/if}
+					</svg>
+				{/if}
+			</div>
+
+			<!-- modals -->
 
 			{#if $geoModalStore.isVisible}
-			<GeoFilterModal
-				{nutsSelectionStore}
-				on:click={toggleGeoModal}
-			/>
+				<GeoFilterModal
+					{nutsSelectionStore}
+					on:click={toggleGeoModal}
+				/>
+			{:else if $infoModalStore.isVisible}
+				<InfoModal
+					{api_doc_url}
+					{api_type}
+					{auth_provider}
+					{data_date}
+					{description}
+					{endpoint_url}
+					{is_public}
+					{is_experimental}
+					{query}
+					{region}
+					{source_name}
+					{source_url}
+					{theme}
+					{url}
+					{warning}
+					{year_extent}
+					on:click={toggleInfoModal}
+				/>
 			{/if}
-		</div>
-
-		<!-- modal -->
-		{#if $infoModalStore.isVisible}
-		<InfoModal
-			{api_doc_url}
-			{api_type}
-			{auth_provider}
-			{data_date}
-			{description}
-			{endpoint_url}
-			{is_public}
-			{query}
-			{region}
-			{source_name}
-			{source_url}
-			{url}
-			{warning}
-			{year_extent}
-			on:click={toggleInfoModal}
-		/>
 		{/if}
-	</section>
+	</div>
 </div>
 
 <style>
 	.time_region_value_IdIndex {
+		display: grid;
+		grid-template-columns: 100%;
+		grid-template-rows: min-content 1fr;
 		height: 100%;
-		width: 100%;
 		user-select: none;
+		width: 100%;
 	}
 
-	header {
-		height: var(--dimHeaderHeight);
-		display: flex;
-		align-items: center;
-		justify-content: space-between;
-	}
-	header div:nth-child(1) {
-		flex: 1;
-	}
-	header div:nth-child(1) h1 {
-		margin: 0;
-	}
-	header div:nth-child(1) p {
-		font-style: italic;
-		font-size: 1rem;
-		color: var(--colorRefLight);
-	}
-	header div:nth-child(2) {
-		align-items: center;
-		cursor: pointer;
-		display: flex;
-		justify-content: space-between;
-		margin-left: 1rem;
-	}
-
-	section {
+	.viewport {
 		display: grid;
-		grid-template-columns: 100%;
-		grid-template-rows: 4rem calc(100% - 4rem);
-		height: calc(100% - var(--dimHeaderHeight));
-		overflow-y: auto;
+		grid-template-rows: 100%;
+		height: 100%;
 		position: relative;
-		width: 100%;
 	}
 
-	.controls {
-		align-items: center;
-		display: flex;
-		height: 100%;
-		justify-content: space-between;
-		width: 100%;
-	}
+	/* small */
 
-	.controls > div:not(:last-child) {
-		margin-right: 0.5rem;
+	.small .viewport {
+		grid-template-columns: 33.333% 33.333% 33.333%;
+		grid-template-areas: 'trends info settings';
+		overflow-y: hidden; /* prevents svg charts from getting taller indefinitely */
+		/* transition:
+			transform
+			var(--transDuration)
+			var(--transFunction); */
```

Comment:
Either a leftover or the comment should be expanded to explain why it's disabled.